network/containerizer/bridgepolicy_integration_test.go

```diff
@@ -110,10 +110,10 @@ func (s *bridgePolicyStateSuite) createSpaceAndSubnet(c *gc.C, spaceName, CIDR s
 	c.Assert(err, jc.ErrorIsNil)
 }
 
-// setupTwoSpaces creates a 'default' and a 'dmz' space, each with a single
-// registered subnet. 10.0.0.0/24 for 'default', and '10.10.0.0/24' for 'dmz'
+// setupTwoSpaces creates a 'somespace' and a 'dmz' space, each with a single
```

Comment:
my apologies for extra changes, we have too many things called default, especially since we now have a default space.